mkosi

```diff
@@ -4429,6 +4434,18 @@ def load_args(args: CommandLineArguments) -> CommandLineArguments:
         elif args.distribution == Distribution.openmandriva:
             args.release = "cooker"
 
+    if args.distribution in (Distribution.centos, Distribution.centos_epel) and \
+        args.release == "8" and \
+        args.output_format == OutputFormat.gpt_btrfs:
```

Comment:
Minor stylistic note: This could be written as 
```python
if (args.distribution in (Distribution.centos, Distribution.centos_epel) and
    args.release == "8" and
    args.output_format == OutputFormat.gpt_btrfs):
```
since the parentheses around the whole condition imply the line breaks, since it's then a single statement. This is, I think more commanly used than explicit line breaks.